### ISSUE TITLE
An error occurred when I used deploy

### ISSUE THREAD

**Issue by athrunsunny** [facebookresearch/detectron2#3551]
Input at terminal ：python export_model.py --config-file  ../../configs/COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml \
    --output ./output --export-method caffe2_tracing --format torchscript \
    MODEL.WEIGHTS detectron2://COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x/137849600/model_final_f10217.pkl \
    MODEL.DEVICE cpu

Tracing the model with torch.jit.trace ...
/home/athrun/anaconda3/envs/detetron/lib/python3.8/site-packages/detectron2/export/c10.py:31: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert tensor.dim() == 2 and tensor.size(-1) in [4, 5, 6], tensor.size()
/home/athrun/anaconda3/envs/detetron/lib/python3.8/site-packages/detectron2/export/c10.py:392: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if num_classes + 1 == class_logits.shape[1]:
/home/athrun/anaconda3/envs/detetron/lib/python3.8/site-packages/detectron2/export/c10.py:401: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  assert box_regression.shape[1] % box_dim == 0
/home/athrun/anaconda3/envs/detetron/lib/python3.8/site-packages/detectron2/export/c10.py:408: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if input_tensor_mode:
/home/athrun/anaconda3/envs/detetron/lib/python3.8/site-packages/detectron2/export/c10.py:440: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  nms_outputs = torch.ops._caffe2.BoxWithNMSLimit(
/home/athrun/anaconda3/envs/detetron/lib/python3.8/site-packages/torch/tensor.py:590: RuntimeWarning: Iterating over a tensor might cause the trace to be incorrect. Passing a tensor of different shape won't change the number of iterations executed (and might lead to errors or silently give incorrect results).
  warnings.warn('Iterating over a tensor might cause the trace to be incorrect. '
/home/athrun/anaconda3/envs/detetron/lib/python3.8/site-packages/detectron2/export/c10.py:469: TracerWarning: Converting a tensor to a Python number might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  for i, b in enumerate(int(x.item()) for x in roi_batch_splits_nms)
/home/athrun/anaconda3/envs/detetron/lib/python3.8/site-packages/detectron2/export/c10.py:76: TracerWarning: Converting a tensor to a Python index might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  data_len = len(value)
/home/athrun/anaconda3/envs/detetron/lib/python3.8/site-packages/detectron2/export/c10.py:95: TracerWarning: Converting a tensor to a Python index might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  return len(self.indices)
/home/athrun/anaconda3/envs/detetron/lib/python3.8/site-packages/torch/jit/_trace.py:958: TracerWarning: Output nr 4. of the traced function does not match the corresponding output of the Python function. Detailed error:
With rtol=1e-05 and atol=1e-05, found 875 element(s) (out of 5582080) whose difference(s) exceeded the margin of error (including 0 nan comparisons). The greatest difference was 2.4378299713134766e-05 (0.5837495923042297 vs. 0.5837252140045166), which occurred at index (12, 27, 5, 23).
  _check_trace(
